Sources/Flows/OAuth2.swift

```diff
@@ -106,11 +106,16 @@ open class OAuth2: OAuth2Base {
 		
 		didAuthorizeOrFail = callback
 		logger?.debug("OAuth2", msg: "Starting authorization")
-		tryToObtainAccessTokenIfNeeded(params: params) { successParams in
+		tryToObtainAccessTokenIfNeeded(params: params) { successParams, error in
 			if let successParams = successParams {
 				self.didAuthorize(withParameters: successParams)
 			}
 			else {
```

Comment:
Add `else if let error = error` here (without the `case .nsError(_)` part – I don't think that's needed).